docs/index.html

```diff
@@ -734,7 +734,7 @@
 </span></button></div></div></div><div class="sc-jSFjdj sc-gKAaRy hsSsLr gcushC"><div class="sc-kYPZxB jdCbTS"><button class="sc-dWBRfb jnEbBv"><span type="get" class="sc-jHcXXw cAOCuf http-verb get">get</span><span class="sc-xGAEC jRjoAh">/milestone/{code}</span><svg class="sc-dIsUp gGvkZD" style="margin-right:-25px" version="1.1" viewBox="0 0 24 24" x="0" xmlns="http://www.w3.org/2000/svg" y="0" aria-hidden="true"><polygon points="17.3 8.3 12 13.6 6.7 8.3 5.3 9.7 12 16.4 18.7 9.7 "></polygon></svg></button><div aria-hidden="true" class="sc-bQCEYZ gBwOdz"><div class="sc-fXgAZx fKFAhr"><div class="sc-iBzEeX sc-cOifOu dFWqin cJyzuM"><p>The production API server.</p>
```

Comment:
No need to run publish or commit changed html
html will be updated automatically on merge

Reply:
fixed

Reply:
and api.yml too